src/locales/en.json

```diff
@@ -257,8 +257,8 @@
     "eula": "End User Legal Agreement",
     "eula_agree_terms_of_use": "I agree that I have read and accepted the terms of use",
     "i_am_over": "Yes, I am over {{minimumAge}}",
-    "location_alert_body": "To use Bluetooth scanning, the device location setting needs to be turned on.",
-    "location_alert_header": "Enable Location access from \"{{applicationName}}\"",
+    "location_alert_body": "To use Exposure Notifications, Location Services must be enabled",
+    "location_alert_header": "Enable Location Services",
```

Comment:
@juancruzgs Can you confirm this copy is correct?

Reply:
Looks good to me 👍